src/app/_modals/nominate-builder/page.tsx

```diff
@@ -0,0 +1,282 @@
+import { headers } from "next/headers";
```

Comment:
This file should no longer exist. this looks like a merge conflict. If you need to make changes to this file you can find it under `@modal/nominate/[walledId]/page`